### PR TITLE
docs(components): [breadcrumb]

### PR DESCRIPTION
**Pull request by jw-foss**
* Update doc file with new syntax.
* Move instances to a dedicated file for readabilities.

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by btea — APPROVED** [review]

File: docs/en-US/component/breadcrumb.md [hunk 1]
```diff
@@ -23,27 +23,31 @@ breadcrumb/icon
...
-| -------------- | -------------------------------- | --------------------- | --------------- | ------- |
-| separator      | separator character              | string                | —               | /       |
-| separator-icon | icon component of icon separator | `string \| Component` | —               | -       |
+### Breadcrumb Attributes
 
-## Breadcrumb Slots
+| Name           | Description                      | Type                     | Accepted Values | Default |
+| -------------- | -------------------------------- | ------------------------ | --------------- | ------- |
+| separator      | separator character              | ^[string]                | —               | /       |
+| separator-icon | icon component of icon separator | ^[string] / ^[Component] | —               | -       |
```
Comment (tolking) [inline docs/en-US/component/breadcrumb.md:33]:
```suggestion
| Name           | Description                      | Type                     | Default |
| -------------- | -------------------------------- | ------------------------ | ------- |
| separator      | separator character              | ^[string]                | /       |
| separator-icon | icon component of icon separator | ^[string] / ^[Component] | -       |
```

File: docs/en-US/component/breadcrumb.md [hunk 2]
```diff
@@ -23,27 +23,31 @@ breadcrumb/icon
...
+### BreadcrumbItem Attributes
 
-| Name    | Description                                               | Type          | Accepted Values | Default |
-| ------- | --------------------------------------------------------- | ------------- | --------------- | ------- |
-| to      | target route of the link, same as `to` of `vue-router`    | string/object | —               | —       |
-| replace | if `true`, the navigation will not leave a history record | boolean       | —               | false   |
+| Name    | Description                                               | Type                                    | Accepted Values | Default |
+| ------- | --------------------------------------------------------- | --------------------------------------- | --------------- | ------- |
+| to      | target route of the link, same as `to` of `vue-router`    | ^[string] / ^[object]`RouteLocationRaw` | —               | ''      |
+| replace | if `true`, the navigation will not leave a history record | ^[boolean]                              | —               | false   |
```
Comment (tolking) [inline docs/en-US/component/breadcrumb.md:48]:
```suggestion
| Name    | Description                                               | Type                                    | Default |
| ------- | --------------------------------------------------------- | --------------------------------------- | --------------- | ------- |
| to      | target route of the link, same as `to` of `vue-router`    | ^[string] / ^[object]`RouteLocationRaw` | ''      |
| replace | if `true`, the navigation will not leave a history record | ^[boolean]                              | false   |
```

File: docs/en-US/component/breadcrumb.md [hunk 3]
```diff
@@ -23,27 +23,31 @@ breadcrumb/icon
...
+| Name           | Description                      | Type                     | Accepted Values | Default |
+| -------------- | -------------------------------- | ------------------------ | --------------- | ------- |
+| separator      | separator character              | ^[string]                | —               | /       |
+| separator-icon | icon component of icon separator | ^[string] / ^[Component] | —               | -       |
+
+### Breadcrumb Slots
 
 | Name | Description               | Subtags         |
 | ---- | ------------------------- | --------------- |
 | -    | customize default content | Breadcrumb Item |
```
[the remark this answers is not in the document]
Comment (tolking) [reply docs/en-US/component/breadcrumb.md:39]: The same goes for below

File: docs/en-US/component/breadcrumb.md (hunk 1 above)
Comment (chenxch) [reply docs/en-US/component/breadcrumb.md:33]: use `—` replace `-`

**Review by tolking — APPROVED** [review]

**Pull request merged**